edit.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link href="toDoApp.css" rel="stylesheet" type="text/css"/>
+    <script src="storage.js"></script>
+    <script src="todoApp.js"></script>
+    <title>Edit Task</title>
+</head>
+<body onload="edit(window.location.search.substr(4))">
```

Comment:
@federob7 : not need to pass this kind of parameters since window is a global object